### PR TITLE
Revised CollisionHandler

### PR DESCRIPTION
**Pull request by adaxz**

### CODE REVIEW

**Comment by imann24** [comment]
@adaxz Have you tested these changes? I don't appear to be able to climb up ladders anymore and the enemy collisions no longer work

**Review by imann24 — CHANGES_REQUESTED** [review]
Verify that collision events still work

File: Assets/Scripts/World/LettersToReid/MyCharacterController.cs [hunk 1]
```diff
@@ -11,7 +11,10 @@ public class MyCharacterController : MonoBehaviour {
     public bool isGrounded;
     public bool canClimb = false;
     
-
+	public Vector3[] memoryLocations = { new Vector3(5.76f, 0.26f, 0), new Vector3(16.21f, 2.37f, 0),
```
Comment (imann24) [inline Assets/Scripts/World/LettersToReid/MyCharacterController.cs]: I still don't think this is quite what we're going for. We shouldn't have to manually tell the system where the memories are. Instead, it should check (similar to the enemy) if the thing it collided with has the tag of "Letter"

Comment (adaxz) [reply Assets/Scripts/World/LettersToReid/MyCharacterController.cs]: Ok. Got your idea.

**Pull request closed without merging**